pyproject.toml

```diff
@@ -42,9 +42,6 @@ build-backend = "pdm.backend"
 [tool.setuptools]
 py-modules = ["laplace"]
 
-[tool.pdm]
```

Comment:
This is because we don't recommend use `pdm` as tool anymore. Note that the `pdm-backend` build backend is separate from `pdm` itself.

Reply:
`uv` has a plan to release its own build-backend. We will replace `pdm-backend` when the time comes.